src/app-logic/url-handling.js

```diff
@@ -366,10 +366,9 @@ export function getQueryStringFromUrlState(urlState: UrlState): string {
         ? undefined
         : urlState.profileSpecific.implementation,
     timelineType:
-      // The default is the category view, so only add it to the URL if it's the
-      // stack or cpu-category view.
-      // TODO: We should make the 'cpu-category' the new default with an upgrader.
-      urlState.profileSpecific.timelineType === 'category'
+      // The default is the cpu-category view, so only add it to the URL if it's
+      // the stack or category view.
+      urlState.profileSpecific.timelineType === 'cpu-category'
         ? undefined
         : urlState.profileSpecific.timelineType,
```

Comment:
I was wondering about not adding it at all if the type is the default (computed) one. But we'd need the profile data, and we don't have it here. Then I'm fine with this.